dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -0,0 +1,105 @@
+import { css } from '@emotion/react';
+import { from, headline, space, textSans } from '@guardian/source-foundations';
+import { decidePalette } from '../lib/decidePalette';
+import { FilterButton } from './FilterButton.importable';
+
+type Props = {
+	availableTopics: Topic[];
+	selectedTopics?: string;
+	format: ArticleFormat;
+	filterKeyEvents: boolean;
+	keyEvents?: Block[];
+	keyEventsId: 'key-events-filter-mobile' | 'key-events-filter-desktop';
+};
+
+const containerStyles = css`
+	padding: ${space[3]}px 0;
+	width: 100%;
+`;
+const headlineStyles = css`
+	${headline.xxxsmall({ fontWeight: 'bold', lineHeight: 'tight' })}
+	padding-bottom: ${space[3]}px;
+`;
+
+const headlineAccentStyles = css`
+	${textSans.small({ fontWeight: 'regular', lineHeight: 'tight' })};
+`;
+
+const betaTextStyles = (palette: Palette) => css`
+	color: ${palette.text.keyEvent};
+`;
+
+const topicStyles = css`
+	display: flex;
+	align-items: flex-start;
+	flex-direction: row;
+	flex-wrap: wrap;
+	column-gap: ${space[2]}px;
+	row-gap: ${space[2]}px;
+	${from.desktop} {
+		flex-direction: column;
+		flex-wrap: nowrap;
+		column-gap: 0;
+	}
+`;
+
+export const TopicFilterBank = ({
+	availableTopics,
+	selectedTopics,
+	format,
+	keyEvents,
+	keyEventsId,
```

Comment:
As mentioned earlier, I think we'd only need this if we wanted to scroll to the button on click - i'd imagine we'd be better of setting `location.hash` to an id on the filter bank or even somewhere at the top of the liveblog maybe?

Reply:
Got you, cheers Joe! I'll remove this